environments/prod/prod.tfvars

```diff
@@ -2245,5 +2245,90 @@ frontends = [
     health_path         = "/"
     health_protocol     = "Https"
     cache_enabled       = "false"
-  }
+  },
+  {
+    product          = "adoption"
+    name             = "adoption"
+    mode             = "Detection"
+    custom_domain    = "apply-for-adoption.platform.hmcts.net"
```

Comment:
you've used adoption.platform.hmcts.net in your other PR https://github.com/hmcts/azure-public-dns/pull/824